src/utilities/FormManager.ts

```diff
@@ -80,20 +79,24 @@ class FormManager {
     };
   }
 
-  public get form () {
+  public get form (): WrappedFormUtils {
     // The form prop continuously changes identity, so we can't just save it locally
     return this.formWrappedInstance.props.form;
   }
 
-  public get fieldConfigs () {
+  public get fieldConfigs (): IFieldConfig[] {
     return getFieldSetsFields(this.args.fieldSets);
   }
 
-  public get submitButtonDisabled () {
+  public get isSubmitButtonDisabled (): boolean {
     return this.hasErrors();
   }
 
-  public getDefaultValue (fieldConfig: IFieldConfig) {
+  public get isCancelButtonDisabled (): boolean {
+    return this.saving;
```

Comment:
in line with other changes, should be `isSaving`?

Reply:
Agreed.